README.md

```diff
@@ -95,6 +95,18 @@ curl -L https://install.meilisearch.com | sh
 
 NB: you can also download Meilisearch from **Homebrew** or **APT** or even run it using **Docker**.
 
+### Import
```

Comment:
What is the point of this section? The import is already explicit in the Getting Started section, right?
Maybe I miss a swift usage.

My point is to avoid having a too "heavy" README, and make it simple and user-friendly, so avoid the duplication of information.

Reply:
Happy to remove or rewrite, the main line from this is trying to call out this:

> When initialising the MeiliSearch client, you must provide a fully qualified URL including scheme.

This has the potential to catch people out, although we do now have some additional protection.

Reply:
I've removed the section in favour of a single line within the Getting Started code sample. Hopefully this is more appropriate for you 🙂